microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Share performance and memory traces
+description: Learn to export and import performance and memory traces in DevTools to get help when investigating issues.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/06/2023
+---
+# Share performance and memory traces
+
+The **Performance** and **Memory** tools record runtime data about your webpage. Exploring the recorded data makes it possible to improve your webpage's memory usage or runtime performance.
+
+Additionally, the recorded data can be exported to files on disk. The exported files are called _traces_.
+
+Exporting performance and memory traces is useful when you want to share these files with other people to get help with investigating issues. An exported trace file can be imported in DevTools at any time.
+
+Traces can be exported in one of two ways: 
+
+* As a normal trace: a `.json`, `.heapsnapshot`, `.heapprofile`, or `.heaptimeline` file, which contains minimal information about the runtime data from the traced webpage.
+
+* As an enhanced trace: a `.devtools` file, which contains more runtime data from the traced webpage.
+
+In Microsoft Edge versions prior to 109, enhanced traces aren't available. Starting with 109, the **Performance** and **Memory** tools can export normal and enhanced traces.
+
+
+<!-- ====================================================================== -->
+## Normal traces
+
+A normal trace only contains some of the information that's present in a webpage. Much of the original code is lost and only the recorded performance or memory data of the webpage is preserved in the trace. For example, when importing a `.heapsnapshot` memory trace that was recorded during another DevTools session or on another computer, there's no way to go from an object in the **Memory** tool to its constructor in the **Sources** tool.
+
+Similarly, in the **Performance** tool; when you import a performance trace recorded earlier, it needs to have been recorded on the same site and same configuration to be able to map from the performance report to the JavaScript that was running in the browser.
+
+When a normal trace is imported in DevTools, the recorded data is used to populate the **Performance** or **Memory** tool.
+
+
+<!-- ====================================================================== -->
+## Enhanced traces
+
+An enhanced trace retains more information about the page. For example, an enhanced trace contains the execution context and the list of parsed scripts, and can optionally contain console messages, script sources, and a snapshot of the DOM tree.
+
+When an enhanced trace is imported in DevTools, a new DevTools window appears. This new window isn't connected to the webpage that's running in your browser, and instead re-creates the environment in which the trace was originally recorded.
```

Comment:
...re-creates *part of* the environment... -- don't want to oversell the feature.